examples/static_map/count_by_key_example.cu

```diff
@@ -71,13 +71,14 @@ __global__ void count_by_key(Map map_view,
   uint64_t thread_unique_keys = 0;
   while (idx < num_keys) {
     // insert key into the map with a count of 1
-    auto [slot, is_new_key] = map_view.insert_and_find({keys[idx], 1});
+    auto [slot, is_new_key] = map_ref.insert_and_find(cuco::pair{keys[idx], 1});
```

Comment:
Side note: It would be nice if we would not have to specify the pair type but that's currently a limitation of the heterogeneous insert implementation. I wonder if it would work if we add an overload `insert_and_find(value_type)`.

Reply:
A nice to have feature. Let me file an issue.

Reply:
#410